x-pack/solutions/observability/plugins/slo/public/components/slo/bulk_purge_confirmation_modal/bulk_purge_confirmation_modal.tsx

```diff
@@ -0,0 +1,166 @@
+        defaultMessage:
+          'Rollup data for {count} SLOs will be purged according to the policy provided below.',
+        values: { count: items.length },
+      })}
+      <EuiSpacer size="m" />
+      <EuiFormRow
+        label={i18n.translate('xpack.slo.bulkPurgeConfirmationModal.purgeTypeLabel', {
+          defaultMessage: 'Purge type',
+        })}
+      >
+        <EuiRadioGroup
+          data-test-subj="sloBulkPurgeConfirmationModalPurgeType"
+          options={[
+            {
+              id: 'fixed_age',
+              label: i18n.translate('xpack.slo.bulkPurgeConfirmationModal.fixedAgeLabel', {
+                defaultMessage: 'Fixed age',
+              }),
+            },
+            {
+              id: 'fixed_time',
+              label: i18n.translate('xpack.slo.bulkPurgeConfirmationModal.fixedDateTimeLabel', {
+                defaultMessage: 'Fixed date/time',
+              }),
+            },
+          ]}
+          idSelected={purgeType}
+          onChange={(val: string) => {
+            setPurgeType(val);
+          }}
+          name="radio group"
+        />
+      </EuiFormRow>
+      {purgeType === 'fixed_age' ? (
+        <EuiFormRow
+          label={i18n.translate('xpack.slo.bulkPurgeConfirmationModal.purgeAgeLabel', {
+            defaultMessage: 'Purge age',
+          })}
+        >
+          <EuiRadioGroup
+            data-test-subj="sloBulkPurgeConfirmationModalPurgeAge"
+            options={[
+              {
+                id: '7d',
+                label: i18n.translate('xpack.slo.bulkPurgeConfirmationModal.last7DaysLabel', {
+                  defaultMessage: 'Last 7 days',
+                }),
+              },
+              {
+                id: '30d',
+                label: i18n.translate('xpack.slo.bulkPurgeConfirmationModal.last30DaysLabel', {
+                  defaultMessage: 'Last 30 days',
+                }),
+              },
+              {
+                id: '90d',
+                label: i18n.translate('xpack.slo.bulkPurgeConfirmationModal.last90DaysLabel', {
+                  defaultMessage: 'Last 90 days',
+                }),
+              },
+            ]}
+            idSelected={age}
+            onChange={setAge}
+          />
+        </EuiFormRow>
+      ) : (
+        <EuiFormRow
+          label={i18n.translate('xpack.slo.bulkPurgeConfirmationModal.purgeDateLabel', {
+            defaultMessage: 'Purge date',
+          })}
+        >
+          <EuiDatePicker showTimeSelect selected={purgeDate} onChange={setPurgeDate} />
+        </EuiFormRow>
+      )}
+      <EuiFormRow>
+        <EuiCheckbox
+          id={basicCheckboxId}
+          checked={forcePurge}
+          onChange={(e) => {
+            setForgePurge(e.target.checked);
+          }}
+          label={i18n.translate('xpack.slo.bulkPurgeConfirmationModal.forcePurge', {
+            defaultMessage: 'Ignore purge policy restrictions',
```

Comment:
"Ignore purge policy restrictions (can delete data belonging to the active SLO window)"